scriptmodules/system.sh

```diff
@@ -473,6 +480,20 @@ function platform_odroid-xu() {
     __platform_flags="arm armv7 neon mali gles"
 }
 
+function platform_jetson-nano() {
+    __default_cpu_flags="-march=armv8-a+crc -mcpu=cortex-a57 -mtune=cortex-a57"
+    __platform_flags+=(aarch64 x11 gl)
+    __default_cflags+=" -ftree-vectorize -funsafe-math-optimizations"
+    __default_asflags=""
+}
+
+function platform_tegra-x1() {
+    __default_cpu_flags="-march=armv8-a+crc -mcpu=cortex-a57 -mtune=cortex-a57"
+    __platform_flags+=(aarch64 x11 gl)
+    __default_cflags+=" -ftree-vectorize -funsafe-math-optimizations -ffast-math -fPIE"
```

Comment:
why -fPIE - I think that probably should be left out.

Also the other cflags should be excluded from here also (See other platforms - we don't include them by default). fast-math could cause incompatibility issues also.

Reply:
I see. I was looking at the odroid and tinker for reference which do have cflags defined, which is why I left them in from the previous users config

Reply:
maybe it would be best to leave out those math based optimizations though and leave enabling those to the cores themselves if they can handle it

Reply:
Yeah. It's safer I think.